modules/artifact/src/main/java/org/mule/runtime/module/artifact/api/classloader/RegionClassLoader.java

```diff
@@ -64,11 +65,14 @@
 public class RegionClassLoader extends MuleDeployableArtifactClassLoader {
 
   protected static final String REGION_OWNER_CANNOT_BE_REMOVED_ERROR = "Region owner cannot be removed";
-
   static {
     registerAsParallelCapable();
   }
 
+
+  private static final Pattern DOT_REPLACEMENT_PATTERN = Pattern.compile("\\.");
+  private static final String PACKAGE_PATH_SEPARATOR = "/";
```

Comment:
Will this work in windows?

Reply:
It should since we are using the normalize function wrapping the generated string for the path. I can also validated in windows afterwards.